src/models/elan1-r1/README.md

```diff
@@ -0,0 +1,29 @@
+# Eland 1U (elan1-r1)
+
+![Eland 1U](./img/elan1-r1-frontports.avif)
+
+The System76 Eland 1U is a rack-mount server with the following specifications:
+
+- Operating System
+    - Ubuntu Server 22.04 LTS (64-bit)
+- CPU options
+    - Supports AMD EPYC™ 7003 series processor family
+    - Single processor
+- Memory
+    - Up to 4096GB (16x256GB) ECC DDR4 @ 3200 MHz
+- Networking
+    - 2x 1GbE LAN ports (1 x Intel® I350-AM2)
+    - 1x 10/100/1000 management LAN
+- Expansion
+    - 1 x PCIe x16 slot (Gen4)
```

Comment:
Can we say `1x PCIe 4.0 x16` to be consistent with Thelio PCIe listings? See mira-b1 for an example.

Reply:
Sounds like a good change